src/main/java/edu/fiuba/algo3/modelo/gestiondeturnos/TurnoFinalizado.java

```diff
@@ -0,0 +1,42 @@
+package edu.fiuba.algo3.modelo.gestiondeturnos;
+
+import edu.fiuba.algo3.modelo.excepciones.EtapaEquivocadaException;
+
+public class TurnoFinalizado implements Turno{
+
+    public Turno avanzarEtapa() {
+        return this;
+    }
+
+    public void atacar(String paisAtaque, String paisDefensa, int cantEjercitos)  {
+        return;
+    }
+
+    public void asignarEjercito(String pais, int cantidad) {
+        return;
+    }
+
+    public void moverEjercito(String paisDesde, String paisHasta, int cantidad) {
+        return;
+    }
```

Comment:
Hay muchos métodos que no la clase no usa, violando el principio de Segregación de Interfaces, separa la interfaz Turno en interfaces más concretas, las anteriores etapas pueden implementar más de una interfaz de ser necesario. Vean donde se usa los métodos comunes y separen según convenga

Reply:
Hola Edon, aca cuando queremos meter un for nos tira "foreach not applicable to type 'java.util.Hashtable<java.lang.String,edu.fiuba.algo3.modelo.distribuciondepaises.Pais>'" marcando "paises" en rojo